snowpack/src/util.ts

```diff
@@ -72,7 +72,16 @@ export function deleteFromBuildSafe(dir: string, config: SnowpackConfig) {
 
 /** Read file from disk; return a string if it’s a code file */
 export async function readFile(filepath: URL): Promise<string | Buffer> {
-  const data = await fs.promises.readFile(url.fileURLToPath(filepath));
+  let data = await fs.promises.readFile(url.fileURLToPath(filepath));
+  if (!data) {
```

Comment:
this is so strange, I can't figure out why this would be returning `undefined`. My best guess is that it's being mocked somewhere in our test suite, but I can't find anything jest related that touches `fs` or `fs.promises`. 

I'm almost certain that this isn't related to these new tests, and is only because we're now testing the `.build()` method via its JS interface.

I really want to diagnose this but I also don't want us to lose a lot of time on it...

Reply:
If you search for `mockName()`, then the optimize plugin is mocking some part of `fs`. That’s likely not being cleaned up properly (we were doing something similar in our webpack test, but that’s since been commented out)

Reply:
I thought so too, but that test is currently being skipped (in the old `plugin-optimize`)